src/ts/constants/chess.ts

```diff
@@ -18,10 +28,23 @@ const PROMOTION_ROLES: RoleDictionary[] = [
   { cgRole: 'bishop', chessRole: BISHOP, urbitRole: PromotionRole.Bishop }
 ]
 
+export interface PieceCount {
+  [key: string]: number;
+}
+
```

Comment:
Should move the `PieceCount` and `PieceValues` types around so it goes typedef A -> const A -> typedef B -> const B.